src/content/deployment/subctl.en.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "subctl"
+weight: 10
+---
+
+`subctl` is a commandline utility designed to simplify the deployment and maintenance of
+submariner across your your clusters.
+
+### Synopsis
+
+`subctl [command] [--flags] ...`
+
+### Description
+
+The subctl program works with the [submariner-operator](https://github.com/submariner-io/submariner-operator)
+it helps automate the operator deployment process reducing the possibility of mistakes during
+the process.
+
+Subctl connects to your cluster(s) and performs the requested *command* operation. For the `join`
```

Comment:
```suggestion
Subctl connects to your cluster(s) and performs the requested *command*. For the `join`
```

I read “requested command” (which works nicely because the synopsis is well constructed), “operation” is redundant.